scripts/setupJest.js

```diff
@@ -87,15 +81,17 @@ global.offeringsStub = {
     twoMonth: {
       offeringIdentifier: 'default',
       product: {
-        intro_price_period_number_of_units: 16,
-        intro_price_period_unit: 'DAY',
-        intro_price_cycles: 1,
-        intro_price_period: 'P2W2D',
-        intro_price_string: '$0.00',
-        intro_price: 0,
+        introPrice: {
```

Comment:
These were all out of date too. We didn't catch it because tests only do a raw comparison.